.github/workflows/tf_apply.yml

```diff
@@ -127,14 +131,15 @@ jobs:
     name: 'Notify Slack'
     runs-on: ubuntu-latest
     needs: [TF-Apply]
+    if: ${{ inputs.notify-slack }}
     steps:
       - uses: ravsamhq/notify-slack-action@v2
         if: always()
         with:
           status: ${{ job.status }}
           token: ${{ secrets.GITHUB_TOKEN }}
           notification_title: "<{repo_url}|{repo}> {workflow}"
-          message_format: "Release ${{ inputs.release-tag }}\ndeployed commit ${{ inputs.commit-identifier }}\n{emoji} ${{ inputs.environment }} {status_message}"
+          message_format: "Release ${{ inputs.release-tag }}\ndeployed commit ${{ inputs.commit-identifier || github.sha }}\n{emoji} ${{ inputs.environment }} {status_message}"
           footer: "Workflow: {run_url}|View"
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.CORE_ENG_DEPLOYMENTS_SLACK_WEBHOOK }}
```

Comment:
If we deleted this step, could we stop passing around the webhook secret? I think people in the team have agreed to remove this step because only Lee looks at [#core-eng-deployments](https://opensesame.slack.com/archives/C02EEGKG8H5)